chase/frontend/app/page.tsx

```diff
@@ -1,12 +1,36 @@
 "use client";
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Image from "next/image";
 import { Button } from "primereact/button";
-import Loading from "./loading";
 import { useRouter } from "next/navigation";
+import { useBackend } from "@/contexts/backend";
+import { useAuth } from "@/contexts/auth";
 
 export default function Home() {
   const router = useRouter();
+  const auth = useAuth();
+  const backend = useBackend();
+
+  const [backendAcceptsOurAuth, setBackendAcceptsOurAuth] = useState(false);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        if (typeof window === "undefined") return;
+        await backend.postApiConferenceCreate({
+          body: {
+            conference: {
+              name: "test conference",
+            },
+            token: "1234",
+          },
```

Comment:
Add TODO to change before production?

Reply:
True, removed the demo code in the newest commit. Thanks!